model/feature_flags.go

```diff
@@ -70,7 +70,8 @@ type FeatureFlags struct {
 
 	CloudFree bool
 
-	CommandPalette bool
+	CommandPalette    bool
+	AdvanceTextEditor bool
```

Comment:
There is a typo, no big deal but usually this things cause confusion down the line.
```suggestion
	AdvancedTextEditor bool
```